File: c++/Program-7.cpp
//Program-7     :- Write a program to find the only non repeating element in a array where every element repeats twice.

#include<iostream>
using namespace std;

int main(){
     int arr[]={2,4,3,5,4,2,5};
     int res=0;
     for(int i=0;i<7;i++){
         res=res^arr[i];
     }

     cout<<res<<"\n";


    return 0;
}

File: java/Program_7.java
public class Program_7 {
    public static void main(String args[]){
  int[] arr={2,3,4,5,5,4,2};
  int res=0;
  for(int i=0;i<arr.length;i++){
      res=res^arr[i];
  }
  System.out.println(res);
    }
}